packs/classfeatures/weapon-expertise.json

```diff
@@ -34,7 +34,8 @@
                         "or": [
                             "class:champion",
                             "class:investigator",
-                            "class:magus"
+                            "class:magus",
+                            "class:swashbuckler"
                         ]
                     }
                 ],
```

Comment:
Is there a reason to predicate this in the first place?

Reply:
Because Sorcerer, Wizard, Oracle, and Witch are also thrown into the mix. I believe this is how they coded for that particular constraint of adding Martial Weapon proficiency to select classes.

Is there a better way of doing this?

Reply:
Ah, I see, the other proficiency bumps are through the details tab, so I didn't see them and thought this was just for the martials. Nevermind me!